.appveyor.yml

```diff
@@ -3,6 +3,8 @@ environment:
       PYTHON: "C:\\Python36"
 
     matrix:
+        - fast_finish: true
```

Comment:
Shouldn't it be outside of the `environment` section?

Reply:
Yeah, of course: https://ci.appveyor.com/project/jazzband/pip-tools/builds/22717305
Fixed.